apps/wallet/src/ui/app/components/accounts/ProtectAccountForm.tsx

```diff
@@ -77,25 +79,30 @@ export function ProtectAccountForm({
 					<SelectField name="autoLockInterval" options={LOCK_INTERVALS} />
 				</div>
 			</div>
+
 			<div className="flex flex-col gap-5 mt-auto">
-				<CheckboxField
-					name="acceptedTos"
-					label={
-						<>
-							I read and agreed to the{' '}
-							<ExternalLink href={ToS_LINK} className="text-[#1F6493] no-underline">
-								Terms of Services
-							</ExternalLink>
-						</>
-					}
-				/>
-				<div className="flex gap-2.5">
-					<Button
-						variant="outline"
-						size="tall"
-						text={cancelButtonText}
-						onClick={() => navigate(-1)}
+				{displayToS ? (
+					<CheckboxField
+						name="acceptedTos"
+						label={
+							<>
+								I read and agreed to the{' '}
+								<ExternalLink href={ToS_LINK} className="text-[#1F6493] no-underline">
```

Comment:
we probably could use Link for this with `href` and I hope the colour and variant is supported already